README_first.ipynb

```diff
@@ -1,27 +1,28 @@
 {
```

Comment:
<p>Those edits look, good but they don't reflect the changes that we have done to the runtime in terms of restructuring.</p><p> </p><p>The <code>get-started</code> and the <code>tutorial-notebooks</code>  folders doesn't exist anymore. Do you think you could revamp this document to match the content of the runtime?</p><p> </p><p>Happy to catch up with you to talk through the user story</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/Graphcore-Tensorflow2-fork/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>